apps/session-manager/api/internal.yaml

```diff
@@ -0,0 +1,805 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Proxy API
+  description: Mobile and web proxy API gateway.
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+security:
+  - Bearer: []
+paths:
+  "/profile":
```

Comment:
Session manager will not expose profile endpoints

Reply:
Fixed in 00b68cf